src/actions/actions.js

```diff
@@ -172,7 +175,7 @@ export function changeCellType(cellType, language = 'js') {
   return (dispatch, getState) => {
     if (isCommandMode(getState())
       && (getSelectedCell(getState()).cellType !== cellType
-      || getSelectedCell(getState()).language !== language)) {
+        || getSelectedCell(getState()).language !== language)) {
```

Comment:
Please make sure your patch doesn't have whitespace only changes like this in it. It makes reviewing the code and understanding the change later much more difficult.

Reply:
Yes I know , It automatically got inserted when I edit some code and save it .

Reply:
I am not able to remove it because when I edit it and save it It automatically move lright. Due to configuration settings in eslint I guess.

Reply:
I don't think this would have to do with eslint. I'd suggest either changing your editor's configuration or switching editors (both atom and vscode would be good for contributing to a project like this).

Reply:
I am using vscode .

Reply:
problem is resolved I am just change my editor configuration and now it is not formatting.